.github/workflows/e2e-test.yml

```diff
@@ -5,18 +5,32 @@ on:
   pull_request:
 
 env:
-  LUNETTES_DOCKERHUB_REPO: local/lunettes
+  REGISTRY: local.io
+  REGISTRY_NAMESPACE: lunettes
 
 jobs:
   create-cluster:
     runs-on: ubuntu-latest
     steps:
+    # Free disk space
+    - name: Free disk space 🧹
```

Comment:
is this icon written here on purpose :D

Reply:
This code snippet is references  the usage of another project [[jupyter/docker-stacks](https://github.com/jupyter/docker-stacks/blob/main/.github/workflows/docker-tag-push.yml#L35)], which also encounters the problem of disk full.